### PR TITLE
billing static implemention

### PR DESCRIPTION
**Pull request by khalifan-kfan**
# Description

Static implementation of billing at the front end.
Note: a few changes from the figma design to follow the app's general design

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Trello Ticket ID

https://trello.com/c/C1OnIVc4

## How Can This Been Tested?

Click on a project and head to billing in the side bar.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots
image

### CODE REVIEW

File: src/components/ProjectBillingPage/ProjectBillingPage.module.css [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  }
+  
+  .MainContentSection {
+    /* divided into two rows */
+    display: grid;
+    grid-template-rows: 80px 4fr;
+    overflow: auto;
+  }
+
+  .SmallContainer{
```
Comment (neelxie) [inline src/components/ProjectBillingPage/ProjectBillingPage.module.css:42]: Have u ever tried, padding: 0 2rem 2rem;

File: src/components/ProjectBillingPage/ProjectBillingPage.module.css [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    background-color: #ffffff;
+    overflow: auto;
+}
+.hr {
+  border-left:1px solid #cccccc;
+}
+.DonutChatContainer{
+    display: flex;
+    flex-direction: column;
+    gap: 0.1;
```
Comment (neelxie) [inline src/components/ProjectBillingPage/ProjectBillingPage.module.css:70]: Did you notice this is not relevant?

File: src/components/ProjectBillingPage/ProjectBillingPage.module.css [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    justify-content: center;
+    border-bottom: 1px solid #cccccc;
+    height: min-content;
+}
+.Heading{
+    padding: .3rem;
+    font-size: large;
+    font-weight: bold;
+}
+.Subtext{
```
Comment (neelxie) [inline src/components/ProjectBillingPage/ProjectBillingPage.module.css:84]: Apart from the padding here, nothing else seems to affect anything.

File: src/components/ProjectBillingPage/ProjectBillingPage.module.css [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+    padding: 0rem 0.6rem;   
+}
+.TotalTxt{
+    text-align: start; 
+}
+.paymentButton{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 0rem 1rem;
```
Comment (neelxie) [inline src/components/ProjectBillingPage/ProjectBillingPage.module.css:140]: padding not being used here.

**Review by neelxie — DISMISSED** [review]
Good work on the implementation.

File: src/components/ProjectBillingPage/ProjectBillingPage.module.css (hunk 1 above)
Comment (khalifan-kfan) [reply src/components/ProjectBillingPage/ProjectBillingPage.module.css:42]: noted and changed

File: src/components/ProjectBillingPage/ProjectBillingPage.module.css [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+      padding-right: 1rem;
+      padding-bottom: 2rem;
+    }
+  }
+  
+.OuterContainerBorder{
+    display: grid;
+    grid-template-columns: 3fr .01fr 4fr;
+    border:1px solid #cccccc;
+    min-width: 0;
```
[the remark this answers is not in the document]
Comment (khalifan-kfan) [reply src/components/ProjectBillingPage/ProjectBillingPage.module.css]: noted and removed

**Review by neelxie — APPROVED** [review]
Good stuff

File: src/components/ProjectBillingPage/ProjectBillingPage.module.css (hunk 4 above)
Comment (khalifan-kfan) [reply src/components/ProjectBillingPage/ProjectBillingPage.module.css]: it has been applied sir

**Review by e-ian — APPROVED** [review]
Great work @khalifan-kfan

**Review by rhodinemma — APPROVED** [review]
nicely done.

**Pull request merged**